plugins/packages/stripe/lib/operations.json

```diff
@@ -1 +1,16 @@
-{}
+{
+    "$schema": "https://raw.githubusercontent.com/ToolJet/ToolJet/develop/plugins/schemas/operations.schema.json",
+    "title": "Stripe datasource",
+    "description": "A schema defining stripe datasource",
+    "type": "api",
+    "defaults": {},
+    "properties": {
+      "operation": {
+        "label": "",
+        "key": "stripe_operation",
+        "type": "react-component-api-endpoint",
+        "description": "Single select dropdown for operation",
+        "specUrl": "https://raw.githubusercontent.com/stripe/openapi/master/openapi/spec3.json"
```

Comment:
This will only work if the deployment has internet connectivity. It will fail if it is deployed in a VPC unless we whitelist this URL. Can we rather include the entire openapi spec here?
We can name the key `openApiSpec`?  Wdyt?

Reply:
Yup

Reply:
It can be done but if you head to the URL, the json is too big so will that be a feasible solution?

Reply:
That is alright